src/stores/AccountController.ts

```diff
@@ -161,6 +161,11 @@ export const getAccount = async (auth: AuthData) => {
   await withAccountService(async ({ accountService, config, accessModel }) => {
     const response = await accountService.getUser({ config, auth });
 
+    const externalData = await accountService.initCustomerExtras();
```

Comment:
If we don't end up creating favorites / history controllers, then I would recommend we add this logic into the Inplayer getUser method instead of adding another service method that is only really used by Inplayer anyhow.

Reply:
good point.